events/_event-landing-macros.html

```diff
@@ -55,53 +59,62 @@
 
    ========================================================================== #}
 
-{% macro event_hero(post) %}
-  {% import "macros/category-icon.html" as category_icon %}
-  <section class="content_hero hero">
-    <div class="hero_wrapper">
-        <!-- TODO replace link with link to event -->
-        <a href="#" class="hero_card hero_card__events">
-            <article class="hero_preview">
-                <span class="category-slug">
-                    <!-- TODO replace temp category with post category. -->
-                    {{ category_icon.render('Announcements & updates', 'category-slug_icon') }}
-                    <span class="u-visually-hidden">Category:</span>
-                    {{ 'Event Category Name' }}
-                </span>
-                <div class="summary">
-                    <header class="summary_header">
-                        <h1 class="summary_heading">
-                            <!-- TODO replace temp title with post title. -->
-                            {{ 'Event Title Here, could possibly break two lines' }}
-                        </h1>
-                    </header>
-                    <p class="summary_text">
-                        <!-- TODO replace temp summary with post summary. -->
-                        {{ 'Include a short summary to give users a quick
-                        overview of the event. Should be less than 200 characters in length.' }}
-                    </p>
-                    <footer class="summary_footer">
-                        <time class="summary_event-date event-meta_datetime dt-start">
-                            <!-- TODO replace temp date/time with post date/time. -->
-                            <span class="event-meta_date">{{ '1/21/2014' }}</span>
-                            <span class="event-meta_time">{{ '2:30 PM CST' }}</span>
-                        </time>
-                    </footer>
-                </div>
-            </article>
-        </a>
-    </div>
-    <div class="hero_img-container hero_img-container__small">
-        <!-- TODO replace background-image with actual image. -->
-        <div class="hero_img"
-             style="background-image:url({{ 'http://placehold.it/660x300.jpg' }});">
+{% macro event_hero(posts) %}
+    {% import "macros/category-icon.html" as category_icon %}
+    {% for post in posts %}
+    {%- set city =  post.venue.address.city + ', ' + post.venue.address.state
+                    | default('Washington, DC') -%}
+    <section class="content_hero hero">
+        <div class="hero_wrapper">
+            <a href="{{ post.permalink }}"
+               class="hero_card hero_card__events">
+                <article class="hero_preview">
+                    {% if post.categories %}
+                    <span class="category-slug">
+                        <!-- TODO replace temp category with post category. -->
```

Comment:
MINOR: include in `{# … #}` instead so it doesn't show in the output on the page.